### PR TITLE
README: Style changes and new examples

### PR DESCRIPTION
**Pull request by DevBonBon**
- Modified some sentences and capitalization.
- Standardized markdown style throughout.
- Moved few paragraphs for clarity
- Added an example for importing using RequireJS / AMD
- Added an example for .attrMap (and event handlers) "Fixes" #31 & #30 

Let me know if there's something you don't like / want changed, and I'll try to figure out how modifying pull requests work :P

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,152 +3,167 @@
...
-You can pass an already available element to crel, and it will be the target of the attributes/child elements
+You can define custom functionality for certain keys seen in the attributes
+object:
 
 ```javascript
-crel(document.body,
-    crel('h1', 'Page title')
-)
+crel.attrMap['on'] = function(element, value) {
+  for (var i in value) {
```
Comment (KoryNunn) [inline README.md]: i should be well named, eg: `eventName`

File: README.md [hunk 2]
```diff
@@ -3,152 +3,167 @@
...
 ```
 
 You can assign child elements to variables during creation:
 
 ```javascript
 var button,
-    wrapper = crel('div',
-        button = crel('button')
-    );
+  wrapper = crel('div', (button = crel('button')));
```
Comment (KoryNunn) [inline README.md]: I'd like to keep this multi-line, as it reads more easily.

File: README.md (hunk 1 above)
Comment (DevBonBon) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (DevBonBon) [reply README.md]: Done

**Comment by KoryNunn** [comment]
Hi, I have made a few changes to your version, but for some reason I can't create a branch that forks off of your master.

I want to make sure you get git-credit for the stuff you've done, so could you please create a branch of of your master, and I'll propose changes against that?

**Comment by KoryNunn** [comment]
Thanks!

**Pull request merged**